utilities/trendQuery.js

```diff
@@ -1,15 +1,16 @@
 const googleTrends = require('google-trends-api');
-const backDateByMonth = require('./backDate');
+const backDateByMonths = require('./backDate');
+const sanitizeTrend = require('./sanitizeTrend');
 
 const trendQuery = (keyword, callback) => {
   const options = {
     keyword: keyword,
-    startTime: backDateByMonth(15),
+    startTime: backDateByMonths(15),
   };
 
   googleTrends.interestOverTime(options)
   .then((results) => {
-    callback(results);
+    callback(sanitizeTrend(JSON.parse(results)));
```

Comment:
Minor comment, no need to change prior to pull.  For sep of concerns I would stick the parse JSON in the sanitizeTrend helper.  No need to recommit over that though.